### ISSUE TITLE
Detect connection of drawing tablet

### ISSUE THREAD

**Issue by basvdheuvel** [troglobit/xplugd#9]
Hi! Many thanks for the great tool.

I have a drawing tablet, and every time I connect it I have to run `xinput` to map it to the correct screen. It would be great if I could do this automatically with `xplugd`.
I'd be willing to add this feature myself, but some pointers would help me a lot.

Thanks!

**Comment by troglobit** [comment]
You should be able to script this without having to modify xplugd.  The man page and example scripts should be a good starting point to get to know xplugd.

**Comment by basvdheuvel** [comment]
Right, I'm already somewhat familiar with configuring xplugd. The problem is that it is not firing when I plug in my drawing tablet. Note that it's not in the output of `xplugd -p`. I'm thinking it's not passing some checks by xplugd to be eligible as e.g. a pointer device.
